MarsRoverBestGroup3.0/MarsRoverBestGroup3.0/APICall.cs

```diff
@@ -38,7 +38,7 @@ public List<Photo> GetMarsRoverPhotosByDateAndRover(DateTime earth_date, string
             var response = _restClient.Get<MarsRoverPhotosResponse>(request);
             if (response.StatusCode != System.Net.HttpStatusCode.OK)
             {
-                throw new Exception("API Call invalid");
+                throw new Exception();
```

Comment:
Can you make this a HttpResponseException instead of just Exception? It takes an argument of the status code which is handy for letting other bits of code know what the actual error was (although we probably don't need to bother with that in this case :) )